src/otaclient/ota_core.py

```diff
@@ -160,8 +160,8 @@ def _download_exception_handler(_fut: Future[Any]) -> bool:
         del exc, _fut  # drop ref to exc instance
 
 
-class _OTAUpdater:
-    """The implementation of OTA update logic."""
+class _OTAUpdateOperator:
+    """The base common class of OTA update logic."""
```

Comment:
This is common(base) class for both client and data updates.
Currently this class only includes proxy, metadata downloading features.